core/module.py

```diff
@@ -205,7 +205,11 @@ def __load_status_vars_activate(self, event):
         # add status vars
         for vname, var in self._stat_vars.items():
             sv = self._statusVariables
-            svar = sv[var.name] if var.name in sv else var.default
+            if isinstance(var.default, (OrderedDict, dict)) and var.name in sv:
```

Comment:
Can be simplified by solely checking for `dict` (`OrderedDict` is subclass of `dict`)